packages/notifications/src/notificationsMiddleware/notificationsMiddleware.test.js

```diff
@@ -3,6 +3,7 @@ import configureStore from 'redux-mock-store';
 import promiseMiddleware from 'redux-promise-middleware';
 import { notificationsMiddleware } from '..';
 import { ADD_NOTIFICATION } from '../redux/actions/action-types';
+import { createPromise } from 'redux-promise-middleware';
```

Comment:
No need to import like this, you are already importing from `promiseMiddleware` on line 3
```JS
import promiseMiddleware, { createPromise } from 'redux-promise-middleware';
```